app/controllers/comments.rb

```diff
@@ -0,0 +1,42 @@
+get '/comments/new' do
+  erb :'comments/new'
+end
+
+post '/comments' do
+  @comment = Comment.new(params[:comment])
+  @suggestion = Suggestion.find(params[:suggestion])
+  @user = User.find(params[:user])
+  @comment.user = current_user
+
+  if @comment.save
+    redirect '/suggestions'
+  else
+    "Adding the comment was unsuccessful, please try again"
+  end
+end
+
+get '/comments/:id' do
+  @comment = Comment.find(params[:id])
+  erb :'comments/show'
+end
+
+get '/comments/:id/edit' do
+  @comment = Comment.find(params[:id])
+  erb :'/comments/edit'
+end
+
+patch '/comments/:id' do
+  @comment = Comment.find(params[:id])
+  @comment.update(params[:comment])
+  if @comment.save
```

Comment:
Protect this update: only the creator of the comment should be able to update it